torch/distributed/_spmd/comm_tensor.py

```diff
@@ -50,6 +51,15 @@ def wrap(work, e):
     return (tree_map(partial(wrap, work), result[0]), work)
 
 
+def _get_tracer(obj: Any) -> Optional[torch.fx.Tracer]:
+    slots = get_proxy_slots(obj)
```

Comment:
there's a global fx tracer table, and that's why we could do this trick?

Reply:
yep, exactly. Below is how `proxy_tensor.py` sets the slots. 

https://github.com/pytorch/pytorch/blob/35f6a69191ef762cf22b6cbfe94b8d9406e16674/torch/fx/experimental/proxy_tensor.py#L55-L58

And it also a similar solution to get the proxy instance. 

https://github.com/pytorch/pytorch/blob/35f6a69191ef762cf22b6cbfe94b8d9406e16674/torch/fx/experimental/proxy_tensor.py#L85-L92